xonsh/xontribs.py

```diff
@@ -202,6 +202,7 @@ def xontribs_load(
     ] = (),
     verbose=False,
     full_module=False,
+    skip_missing=False,
```

Comment:
could you name this to silence or something similar?

Reply:
it doesn't signify what it is doing. By default it is going to skip loading xontribs not installed but will show warning that is not installed. So we have to make it clear that we are silencing those warnings shown and no other behaviour is changed.

Reply:
how about `no_warn_missing` ?

Reply:
that's a bit of an anti-pattern -- better to have the name be `warn_missing` and then check for `if not warn_missing` or similar

Reply:
We can decide what the name of an `argparse` argument is, and what the name of the variable it targets is -- if the automatic argparse thing can't do that, we should just use straight argparse for this.

Reply:
@anki-code, how about `suppress_warnings` or `silence_warnings`?